mllm/backends/qnn/aot/QnnTargetMachine.hpp

```
@@ -0,0 +1,67 @@
+  kHtpHighPowerSaver,
+  kHtpLowBalanced,
+  kHtpBalanced,
+};
+
+//  Protection Domain Session
+enum QcomSecurityPDSession : uint32_t {
+  kHtpUnsignedPd = 0,
+  kHtpSignedPd,
+};
+
+struct QcomTargetMachine {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: soc_htp_chipset, soc_htp_arch, soc_htp_performance, soc_htp_security_pd_session, soc_htp_vtcm_total_memory_size

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)